FirebaseRemoteConfig/Tests/SwiftAPI/APITests.swift

```diff
@@ -18,10 +18,24 @@ import FirebaseCore
 import XCTest
 
 class APITests: APITestBase {
+  var console: RemoteConfigConsole!
+
   override func setUp() {
     super.setUp()
     if APITests.useFakeConfig {
       fakeConsole.config = ["Key1": "Value1"]
+    } else {
```

Comment:
Shouldn't there be three cases here?  1. fakeConsole 2. RemoteConfigConsole() 3. Real console.

Reply:
Yes, you're right!